packages/yew/src/virtual_dom/mod.rs

```diff
@@ -368,10 +368,10 @@ pub(crate) trait VDiff {
     ) -> NodeRef;
 }
 
-pub(crate) fn insert_node(node: &Node, parent: &Element, next_sibling: Option<Node>) {
+pub(crate) fn insert_node(node: &Node, parent: &Element, next_sibling: &Option<Node>) {
```

Comment:
```suggestion
pub(crate) fn insert_node(node: &Node, parent: &Element, next_sibling: Option<&Node>) {
```
Taking `Option` itself as reference instead of the underlying data feels wrong

Reply:
Hmm, agreed.